modules/reloadBidAdapter.js

```diff
@@ -0,0 +1,513 @@
+
+      var vPrxClientTool = new ReloadClientTool({
+        prxVer: VERSION_ADAPTER,
+        prxType: "bd",
+
+        plcmID: bidRequest.params.plcmID,
+        partID: bidRequest.params.partID,
+        opdomID: bidRequest.params.opdomID,
+        bsrvID: bidRequest.params.bsrvID
+      });
+
+      var vImpression = {
+        // Data from BidRequest. id only required
+        id: bidRequest.bidId,
+        bidId: bidRequest.bidId,
+        adUnitCode: bidRequest.adUnitCode,
+        transactionId: bidRequest.transactionId,
+        bidderRequestId: bidRequest.bidderRequestId,
+        auctionId: bidRequest.auctionId,   
+        
+        banner: {
+          h: vDim[1],
+          w: vDim[0],
+          ext: {
+            type: bidRequest.params.type || 'pcm', 
+            pcmdata: vPrxClientTool.getPCMObj()
+          }
+        }
+      };
+      bidReq.imp.push(vImpression);
+
+    }
+    
+    if (bidReq.imp.length>0) {
+      const payloadString = JSON.stringify(bidReq);
+      vRequests.push({
+        method: 'POST',
+        url: vPrxClientTool.getSrvUrl(),
+        data: payloadString
+      });
+    }
+    return vRequests;
+  },
+  /**
+   * Unpack the response from the server into a list of bids.
+   *
+   * @param {ServerResponse} serverResponse A successful response from the server.
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function (serverResponse, bidRequest) {
+    const serverBody = serverResponse.body;
+
+    const bidResponses = [];
+
+    for (var vIdx = 0; vIdx < serverBody.seatbid.length; vIdx++) {
+      var vSeatBid = serverBody.seatbid[vIdx];
+
+      for (var vIdxBid = 0; vIdxBid < vSeatBid.bid.length; vIdxBid++) {
+        var vBid = vSeatBid.bid[vIdxBid];
+
+        var vPrxClientTool = new ReloadClientTool({
+          plcmID: vBid.ext.plcmID,
+          partID: vBid.ext.partID,
+          opdomID: vBid.ext.opdomID,
+          bsrvID: vBid.ext.bsrvID
+        });
+
+        vPrxClientTool.setPCMObj(vBid.ext.pcmdata);
+
+        if (vPrxClientTool.getBP() > 0) {
+          var bidResponse = {
+            requestId: vBid.impid,
+            ad: vPrxClientTool.getAM(),
+            cpm: vPrxClientTool.getBP() / 100,
+            width: vBid.ext.banner.w,
+            height: vBid.ext.banner.h,
+            creativeId: vBid.id,
+            currency: vPrxClientTool.getBC(),
+            ttl: 300,
+            netRevenue: true
+          };
+          bidResponses.push(bidResponse);
+        }
+      }
+    }
+
+    return bidResponses;
+  },
+
+  /**
+   * Register the user sync pixels which should be dropped after the auction.
+   *
+   * @param {SyncOptions} syncOptions Which user syncs are allowed?
+   * @param {ServerResponse[]} serverResponses List of server's responses.
+   * @return {UserSync[]} The user syncs which should be dropped.
+   */
+  getUserSyncs: function (syncOptions, serverResponses) {
```

Comment:
Please remove if you are not using getUserSyncs

Reply:
Fixed